design/use_cases/indoor_navigation_use_case.md

```diff
@@ -0,0 +1,14 @@
+# Indoor Navigation
+As a Robot user I want my robot to autonomously navigate to a given location on a given map so that it can help me at that location
+
+## More details
+- Why is this needed?
+   - This is needed for indoor robot navigation in most (all?) cases
+   - Example: a courier robot in a logistics warehouse
```

Comment:
Something to consider -- RE: indoor vs. outdoor as well: ramps are non-planar. They are poorly supported in today's system -- yet they are still indoor. The dividing line really is "all 2d planar environment" vs "environment that has non-planar ground". The efficiencies of the 2d planar case really helps for robots like Turtlebot, where there isn't a huge processing power onboard.

Reply:
I tried to cover 2D+ separately in a different use case. I didn't specifically mention ramps though, but I did mention stairs and elevators. I think ramps belong in that use case also. I can add them there.